### PR TITLE
Move package to non-preview, add changelog notes

### PR DESCRIPTION
**Pull request by rjmholt**
~~Just waiting on @SydneyhSmith's changelog link~~

Ready

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,10 +1,10 @@
 {
-  "name": "powershell-preview",
-  "displayName": "PowerShell Preview",
+  "name": "powershell",
+  "displayName": "PowerShell",
   "version": "2020.3.0",
   "preview": true,
```
Comment (TylerLeonhardt) [inline package.json]:
```suggestion
  "preview": false,
```

**Comment by TylerLeonhardt** [comment]
wait

**Comment by TylerLeonhardt** [comment]
is this the right branch? This should be `release/stable/2020.3.0`?

**Comment by rjmholt** [comment]
> his should be release/stable/2020.3.0?

There's no `stable` branch set. There's `release/<version` and `release/preview/<version>`. We could change that though

**Comment by TylerLeonhardt** [comment]
I know in the release repo we do `stable` and `preview`... maybe we should start following that?

**Pull request merged**